packages/eden/src/path-params.ts

```diff
@@ -56,5 +56,5 @@ export type ExtractEdenTreatyRouteParams<T> = {
  * @see https://elysiajs.com/eden/treaty/overview.html#dynamic-path
  */
 export type ExtractEdenTreatyRouteParamsInput<T> = {
-  [K in keyof T as K extends `:${infer TParam}` ? TParam : never]: string | number
+  [K in keyof T as K extends `:${infer TParameter}` ? TParameter : never]: string | number
```

Comment:
I don't really think it's worth it to fully verbalize internal type generics because they're not really supposed to be viewed by anybody except maintainers. And `TParameter` sounds like it's related to a function parameter, but this type has nothing to do with functions.